data/json/recipes/practice/melee.json

```
@@ -149,39 +149,487 @@
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "great hammers (intermediate)",
+    "description": "Practice advanced methods of using an oversized hammer in combat.",
+    "skill_used": "bashing",
+    "skills_required": [ [ "melee", 1 ] ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 2, "max_difficulty": 3, "skill_limit": 4 },
+    "autolearn": [ [ "bashing", 4 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_great_hammers_familiar", "required": true },
+      { "proficiency": "prof_great_hammers_pro", "time_multiplier": 2, "skill_penalty": 0 }
+    ],
+    "tools": [
+      [ "warhammer", "hammer_sledge", "hammer_sledge_heavy" ],
+      [ "pseudo_training_dummy_light", "pseudo_training_dummy_heavy" ]
+    ]
+  },
+  {
+    "id": "prac_bashing_beg_flail",
+    "type": "practice",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "flails (beginner)",
+    "description": "Practice the basics of using a flail in combat.",
+    "skill_used": "bashing",
+    "skills_required": [ [ "melee", 1 ] ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_limit": 2 },
+    "autolearn": [ [ "bashing", 2 ] ],
+    "proficiencies": [ { "proficiency": "prof_flails_familiar", "time_multiplier": 1.5, "skill_penalty": 0 } ],
+    "tools": [ [ "2h_flail_wood", "2h_flail_steel" ] ]
+  },
+  {
+    "id": "prac_bashing_int_flail",
+    "type": "practice",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "flails (intermediate)",
+    "description": "Practice advanced methods of using a flail in combat.",
+    "skill_used": "bashing",
+    "skills_required": [ [ "melee", 1 ] ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 2, "max_difficulty": 3, "skill_limit": 4 },
+    "autolearn": [ [ "bashing", 4 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_flails_familiar", "required": true },
+      { "proficiency": "prof_flails_pro", "time_multiplier": 2, "skill_penalty": 0 }
+    ],
+    "tools": [ [ "2h_flail_wood", "2h_flail_steel" ], [ "pseudo_training_dummy_light", "pseudo_training_dummy_heavy" ] ]
+  },
+  {
+    "id": "prac_bashing_beg_mace",
+    "type": "practice",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "maces (beginner)",
+    "description": "Practice the basics of using a mace or a club in combat.",
+    "skill_used": "bashing",
+    "skills_required": [ [ "melee", 1 ] ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_limit": 2 },
+    "autolearn": [ [ "bashing", 2 ] ],
+    "proficiencies": [ { "proficiency": "prof_maces_familiar", "time_multiplier": 1.5, "skill_penalty": 0 } ],
+    "tools": [ [ "club_wooden", "mace", "mace_simple", "morningstar" ] ]
```

```suggestion
    "tools": [ [ "club_wooden", "mace", "mace_trench", "morningstar" ] ]
```
(or have both the sturdy & non-sturdy versions of trench mace as options)

---

Both study and non-sturdy seems prudent